### PR TITLE
Initial Setup for Component Library

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Added basic landing page
- Added Documentation Page with sections 
      a. Installation
      b. Customization
      c. Colors
      d. Typography (Text Utilities)
- Added CSS folder structure for the Library
- Added necessary css files.
- Added logo

Landing Page

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Ui looks so good .the best one i have reviewed till yet

**Comment by pritam-kr** [comment]
Wow, The best one seen till now.

**Review by pritam-kr — APPROVED** [review]

File: styles/components/componentColor.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+.infoDark {
+  background-color: var(--infoDark);
+  color: whitesmoke;
+}
+
+.defaultDark {
+  background-color: var(--defaultDark);
+  color: whitesmoke;
```
Comment (prathmeshjagtap) [inline styles/components/componentColor.css]: you can use the variable naming  for WhiteSmoke also you have used this color at many palace

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

**Comment by SanjayJatti** [comment]
You can create variable for shadow also to be more consistent.
Otherwise, It is looking amazing. Kudos to you.

**Review by SanjayJatti — APPROVED** [review]

File: styles/components/componentColor.css (hunk 1 above)
Comment (SukanyaSen530) [reply styles/components/componentColor.css]: fixed - 365617acdd786a7cff4439ba15f4fa06aedb7694

**Review by prathmeshjagtap — APPROVED** [review]

**Pull request merged**